adserver/templates/adserver/reports/publisher.html

```diff
@@ -14,6 +14,18 @@
   <li class="breadcrumb-item active">{{ publisher }}</li>
 {% endblock breadcrumbs %}
 
+{% block additional_filters %}
```

Comment:
Is the reasoning behind this to see non-paid campaigns for specific publishers? This filter is exposed to publishers. Do we think that's important to them?

Reply:
Yea, at least for now I'm curious to see the paid vs. non-paid views. But maybe once we're fully spun up and turn `paid ads only` on, it won't be important?

Reply:
I think it's fine for now. I suspected that most publishers would want to restrict ads shown on their property to paid only. However, maybe that isn't the case. Maybe rather than `Publisher.paid_ads_only` we need multiple options to allow paid, allow house, etc.